Packs/URLHaus/ReleaseNotes/1_0_37.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### URLhaus
+
+- Starting June 30th 2025, abuse.ch is requiring a new authentication method with 'Auth-Key' header. 
+'Auth-Key' param is mandatory.
```

Comment:
Just replace with something like "Fixed an issue where the pack couldn't update when there was a configured instance."